Jenkinsfile

```diff
@@ -69,11 +71,28 @@ pipeline {
                 }
             }
         }
+        stage("Build examples' images for testing"){
+            steps {
+                script {
+                    dir ("${WORKING_DIR}") {
+                        // Do not build native images for the PR checks
+                        sh "make build-examples-images tags='~@native' concurrent=1 ${getBDDParameters('never', false)}"
```

Comment:
It also needs `cr_deployment_only=true`, though it can be added directly in makefile.....
Will do so.

Reply:
Well, actually not needed here as CLI is build before.

Reply:
https://github.com/kiegroup/kogito-cloud-operator/pull/458